frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/capacity-card/capacity-card.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+
+import * as plugins from '../../../../../../../public/plugins';
+import {
```

Comment:
```suggestion
import { DashboardCard } from '@console/internal/components/dashboard/dashboard-card/card';
```

Reply:
Use the same way to import all other internal imports.

Reply:
Thanks Ankush, addressed in the latest PR...